package.json

```diff
@@ -55,7 +55,7 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap --use-workspaces",
-    "build": "lerna exec --parallel -- babel --root-mode upward src -d lib --ignore **/*.story.js,**/*.spec.js",
+    "prepublish": "lerna exec --parallel -- babel --root-mode upward src -d lib --ignore **/*.story.js,**/*.spec.js",
```

Comment:
I actually like this because this is triggered every time `yarn` is ran. As a result, one can be sure that all the packages have been built after installing the node_modules in the development environment.